packages/patternfly-4/react-core/src/components/Tabs/TabContent.d.ts

```diff
@@ -0,0 +1,9 @@
+import { FunctionComponent, HTMLProps } from 'react';
+
+export interface TabContentProps extends HTMLProps<HTMLDivElement> {
+  id: string;
```

Comment:
I think `id` of type string might already part of the interface you're extending from
`interface HTMLAttributes` extended by `interface AllHTMLAttributes` extended by `interface HTMLProps` which you're extending
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1432

Reply:
Yup, you're right.